markup_fmt/tests/fmt/jinja/single-tag/fixture.jinja

```diff
@@ -1,4 +1,17 @@
+
```

Comment:
I guess this empty line should be unexpected.

Reply:
Absolutely, was me doing some debugging on the first line x)